[Documentation] Update Invoke your skill documentation for C# (#477)
southworks/botframework-solutions: docs/_docs/skills/tutorials/create-skill/csharp/8-invoke-your-skill.md

## {{ page.title }}

In order to confirm that the skill connection with the Virtual Assistant was correctly made, you can test the communication using the **Bot Framework Emulator**, sending an utterance from the Virtual Assistant that should be recognized by the Skill (e.g., "Run sample dialog"). You will see how the Skill starts the workflow, and when the skill is finished it will send the control back to the Virtual Assistant.

1. Press **F5** within Visual Studio to run your Virtual Assistant.
1. Open the **Bot Framework Emulator**.
1. Select **Open Bot**.

    ![]({{site.baseurl}}/assets/images/quickstart-virtualassistant-openbot.png)

> 1. Provide the messaging endpoint of your running Virtual Assistant (e.g: http://localhost:3978/api/messages), and the Microsoft App ID and Microsoft App Password values from its `appsettings.json` file.

Reviewer: ```suggestion
1. Provide the messaging endpoint of your running Virtual Assistant (e.g: http://localhost:3978/api/messages), the Microsoft App ID and Microsoft App Password values from its `appsettings.json` file.
```

## {{ page.title }}

> In order to confirm that the skill connection with the Virtual Assistant was correctly made, you can test the communication using the **Bot Framework Emulator**, sending an utterance from the Virtual Assistant that should be recognized by the Skill (e.g., "Run sample dialog"). You will see how the Skill starts the necessary workflow, and when the skill is finished it will send the control back to the Virtual Assistant.

Reviewer: ```suggestion
In order to confirm that the skill connection with the Virtual Assistant was correctly made, you can test the communication using the **Bot Framework Emulator**, sending an utterance from the Virtual Assistant that should be recognized by the Skill (e.g., "Run sample dialog"). You will see how the Skill starts the workflow, and when the skill is finished it will send the control back to the Virtual Assistant.
```

> You can invoke your Skill using these steps:

Reviewer: ```suggestion
You can invoke your Skill following these steps:
```

> 
> You can invoke your Skill following these steps:

Reviewer: ```suggestion
```

Author: Do you think we should remove that line?

Reviewer: Yes! I think that showing the enumered list, the user infers that are steps

1. Press **F5** within Visual Studio to run your Virtual Assistant.
1. Open the **Bot Framework Emulator**.
1. Select **Open Bot**.

    ![]({{site.baseurl}}/assets/images/quickstart-virtualassistant-openbot.png)

> 1. Provide the messaging endpoint of your running bot (e.g: http://localhost:3978/api/messages).

Reviewer: ```suggestion
1. Provide the messaging endpoint of your running Virtual Assistant (e.g: http://localhost:3978/api/messages).
```

> 1. Provide the Microsoft App ID and Microsoft App Password values from your appsettings.json file.

Reviewer: ```suggestion
1. Provide the Microsoft App ID and Microsoft App Password values from the Virtual Assistant's `appsettings.json` file.
```

> 1. Provide the Microsoft App ID and Microsoft App Password values from the Virtual Assistant's `appsettings.json` file.

Reviewer: I think that we can merge these two steps to only one.

    ![]({{site.baseurl}}/assets/images/quickstart-virtualassistant-openbotmodal.png)

> 1. The greeting card of your **Virtual Assistant** should appears.

Reviewer: ```suggestion
1. The greeting card of your **Virtual Assistant** should appear.
```

> 1. Write your name, and then invoke your skill writing *"Run sample dialog"*.

Reviewer: ```suggestion
1. Write your name, and invoke your skill writing *"Run sample dialog"*.
```

    ![]({{site.baseurl}}/assets/images/virtualAssistant-Skill-communication.png)

> 1. Congratulations, you’ve invoked your skill throught your Virtual Assistant!

Reviewer: ```suggestion
1. Congratulations, you’ve invoked your skill through your Virtual Assistant!
```
